### DISCUSSION TITLE
[UDP] dial 8.8.8.8:53: client handshake: UDP ASSOCIATE: command not supported"

### DISCUSSION THREAD

**Discussion by 4rt3mT** [xjasonlyu/tun2socks#135] [Q&A]
Why when i try to set dns server in console tun2socks print:

`[UDP] dial 8.8.8.8:53: client handshake: UDP ASSOCIATE: command not supported"`

without tun2socks works normally

**Comment by xjasonlyu** [comment]
I think the log is pretty much clear that indicates your server doesn’t support UDP.

**Reply by 4rt3mT** [reply]
Without dns UDP works correctly

**Reply by xjasonlyu** [reply]
Don’t you have a socks5 server for it?

**Reply by 4rt3mT** [reply]
Its socks5 with login:pass

**Reply by 4rt3mT** [reply]
But yes, all UDP dont come through, why?

**Reply by 4rt3mT** [reply]
I mean, how proxy with login pass dont support UDP, is this real?

**Reply by 4rt3mT** [reply]
I've tried more socks5 proxy, all of these says

`[UDP] dial 8.8.8.8:53: client handshake: UDP ASSOCIATE: command not supported"`

**Reply by xjasonlyu** [reply]
this error means your socks5 server doesn’t support UDP relay, try to use a valid socks5 server with UDP support.

**Reply by 4rt3mT** [reply]
It is 100% proxy error, not mine?

**Reply by xjasonlyu** [reply]
nope, it’s your server side issue.

**Reply by 4rt3mT** [reply]
Thank you
